client/src/styling.jsx

```diff
@@ -2,29 +2,45 @@ import styled from 'styled-components';
 
 export const Main = styled.section`
   padding: 1em;
-  background: papayawhip;
+  background: rgb(250,250,250);
   max-width: 500px;
   font-family: Arial, Helvetica, sans-serif;
   flex-direction: row;
 `;
 
 // Component Containers
 
-export const NamePriceContainer = styled.section`
+export const NamePriceContainer = styled.div`
   padding: 0.5em;
   background: white;
+  background: rgb(250,250,250)
```

Comment:
Shouldn't we get rid of the first `background: white` declaration?

Reply:
Good eye!